website/content/en/docs/main/tasks/private-networks/index.md

```diff
@@ -0,0 +1,394 @@
+    metadata:
+        name: clusterlink-frps
+        namespace: clusterlink-system
+    spec:
+        type: NodePort
+        selector:
+            app: frps
+        ports:
+          - port: 4443
+            targetPort: 4443
+            nodePort: 30444
+    " | kubectl apply -f -
+    ```
+
+    In this case, we use a `NodePort` service, but it can be other types like `LoadBalancer`.
+
+## Create FRPs Clients
+
+1. Set the `FRP_SERVER_IP` variable for each cluster:
+
+    *Client cluster*:
+
+    ```sh
+    export FRP_SERVER_IP=`docker inspect -f '{{range .NetworkSettings.Networks}}{{.IPAddress}}{{end}}' client-control-plane`
+    ```
+
+    *Client cluster*:
+
+    ```sh
+    export FRP_SERVER_IP=`docker inspect -f '{{range .NetworkSettings.Networks}}{{.IPAddress}}{{end}}' client-control-plane`
+    ```
+
+2. Deploy FRP client configuration on each cluster:
+
+    *Client cluster*:
+
+    ```sh
+    echo "
+    apiVersion: v1
+    kind: ConfigMap
+    metadata:
+        name: frpc-config
+        namespace: clusterlink-system
+    data:
+        frpc.toml: |
+            # Set server address
+            serverAddr = \""${FRP_SERVER_IP}"\"
+            serverPort = 30444
+
+            [[proxies]]
+            name = \"clusterlink-client\"
+            type = \"stcp\"
+            localIP = \"clusterlink.clusterlink-system.svc.cluster.local\"
+            localPort = 443
+            secretKey = \"abcdefg\"
+
+            [[visitors]]
+            name = \"clusterlink-client-to-server-visitor\"
+            type = \"stcp\"
+            serverName = \"clusterlink-server\"
+            secretKey = \"abcdefg\"
+            bindAddr = \"::\"
+            bindPort = 6002
+    " | kubectl apply -f -
+    ```
+
+    *Server cluster*:
+
+    ```sh
+    echo "
+    apiVersion: v1
+    kind: ConfigMap
+    metadata:
+        name: frpc-config
+        namespace: clusterlink-system
+    data:
+        frpc.toml: |
+            # Set server address
+            serverAddr = \""${FRP_SERVER_IP}"\"
+            serverPort = 30444
+
+            [[proxies]]
+            name = \"clusterlink-server\"
+            type = \"stcp\"
+            localIP = \"clusterlink.clusterlink-system.svc.cluster.local\"
+            localPort = 443
+            secretKey = \"abcdefg\"
+
+            [[visitors]]
+            name = \"clusterlink-server-to-client-visitor\"
+            type = \"stcp\"
+            serverName = \"clusterlink-client\"
+            secretKey = \"abcdefg\"
+            bindAddr = \"::\"
+            bindPort = 6001
+    " | kubectl apply -f -
+    ```
+
+    For each configuration, we first set the FRP server IP and port number.
+    We create a `proxy` that connects to the ClusterLink gateway and establishes a reverse tunnel to allow other clients to connect.
+    We also create an FRP `visitor` that specifies which other peers this client wants to connect to (you need to create a visitor for every peer you want to connect).
```

Comment:
do you want to put a reference to the FRP configuration file format on their website?
Otherwise, it's quite mysterious all these sections...
Or you could add 1 sentence description of each section (`proxy` section is used to define... Similarly, the `visitors` section defines...)